lib/geordi/docker.rb

```diff
@@ -28,11 +31,31 @@ def shell(options = {})
     end
 
     def vnc
-      Cucumber.new.launch_vnc_viewer('::5967')
+      check_installation_and_config
+      launch_vnc_viewer('::5967')
     end
 
+
     private
 
+    def launch_vnc_viewer(source = VNC_DISPLAY)
+      fail('VNC viewer not found. Install it with `geordi docker vnc --setup`.') unless command_exists?('vncviewer')
+
+      fork do
+        error = capture_stderr do
+          system("#{VNC_VIEWER_COMMAND} #{source}")
+        end
+        unless $?.success?
+          if $?.exitstatus == 127
+            fail('VNC viewer not found. Install it with `geordi docker vnc --setup`.')
+          else
+            fail("VNC viewer could not be opened: #{error}")
+          end
+        end
+      end
+      success('Launching VNC viewer was successful.')
```

Comment:
`success` is intended as a conclusion of an action that was `announce`d. Since the opened VNC window proves success, please simply remove this line.

Reply:
In order to test the `geordi docker vnc` command, I added this success message. The output would not contain any feedback without this, but I can replace that with a puts if that is more fitting?

Reply:
This should help you without a success message: https://relishapp.com/cucumber/aruba/v/0-11-0/docs/command/check-exit-status-of-commands